fluidplayer.js

```diff
@@ -4253,7 +4253,8 @@ var fluidPlayerClass = {
             var containerDiv = document.createElement('div');
             containerDiv.id = this.videoPlayerId + '_fluid_pseudo_poster';
             containerDiv.className = 'fluid_pseudo_poster';
-            containerDiv.style.background = "url('" + this.displayOptions.layoutControls.posterImage + "') center center / contain no-repeat black";
+            containerDiv.style.background = "url('" + this.displayOptions.layoutControls.posterImage + "') center center / "
+                + this.displayOptions.layoutControls.posterImageSize + " no-repeat black";
```

Comment:
Maybe worth adding some validation for posterImageSize values to prevent people adding possible garbage here.

Reply:
Yeap, sure. I will add.

Reply:
Look now